synfig-studio/src/gui/cellrenderer/cellrenderer_value.cpp

```diff
@@ -402,10 +402,20 @@ CellRenderer_ValueBase::render_vfunc(
 			Distance x( vector[0], Distance::SYSTEM_UNITS ), y( vector[1], Distance::SYSTEM_UNITS );
 			x.convert( App::distance_system, get_canvas()->rend_desc() );
 			y.convert( App::distance_system, get_canvas()->rend_desc() );
-			property_text() = strprintf("%s,%s", x.get_string(real_num_decimals).c_str(), y.get_string(real_num_decimals).c_str());
+			std::string text = strprintf("%s,%s", x.get_string(real_num_decimals).c_str(), y.get_string(real_num_decimals).c_str());
```

Comment:
It looks like it's better to add a flag to `Distance::get_string(int digits, bool remove_trailing_zeroes)`

Reply:
sounds like a good idea

Reply:
done